ci-operator/step-registry/upi/install/vsphere/upi-install-vsphere-commands.sh

```diff
@@ -73,7 +73,7 @@ function gather_bootstrap_and_fail() {
     GATHER_BOOTSTRAP_ARGS+=('--master' "${control_plane_0_ip}" '--master' "${control_plane_1_ip}" '--master' "${control_plane_2_ip}")
 
     set -e
-    openshift-install --dir=/tmp/artifacts/installer gather bootstrap --key "${SSH_PRIV_KEY_PATH}" "${GATHER_BOOTSTRAP_ARGS[@]}"
+    openshift-install --dir="${ARTIFACT_DIR}" gather bootstrap --key "${SSH_PRIV_KEY_PATH}" "${GATHER_BOOTSTRAP_ARGS[@]}"
```

Comment:
This will end up with an .openshift_install.log that is out of order. The first part of the log file will contain the logs from `gather bootstrap`. Then the rest of the log file will contain the logs from `wait-for bootstrap-complete`.